pkg/agent/cniserver/pod_configuration.go

```diff
@@ -416,7 +416,14 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod, containerAccess *contain
 		desiredPods.Insert(k8s.NamespacedName(pod.Namespace, pod.Name))
 	}
 
+	missingIfConfigs := make([]*interfacestore.InterfaceConfig, 0)
 	for _, containerConfig := range knownInterfaces {
+		// Find the OVS ports which are not connected to host interfaces, this is useful on Windows if the runtime is
+		// containerd, because the host interface is created async from the OVS port.
+		if containerConfig.OFPort == -1 {
+			missingIfConfigs = append(missingIfConfigs, containerConfig)
+			continue
+		}
```

Comment:
By skipping the following process, interfaces that belong to deleted pods will not be deleted.